### PR TITLE
Localization consts for Request Call Page

### PR DESCRIPTION
**Pull request by jasperhuangg**
This PR adds some translations for text that'll be needed when implementing the Request Call Page.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/167900

### Tests
N/A

### QA Steps
N/A

### Tested On
N/A

### CODE REVIEW

**Comment by jasperhuangg** [comment]
@HorusGoul requesting you for review specifically for a quick double-check on the Spanish translations, thanks!

File: src/languages/es.js [hunk 1]
```diff
@@ -356,7 +356,7 @@ export default {
         requestACall: 'Llámame por teléfono',
         description: 'Necesitas ayuda configurando tu cuenta? Nuestro equipo de guías te pueden ayudar.',
         instructions: 'Escribe tu nombre y número de teléfono y te llamaremos.',
-        availabilityText: '*Nuestros guías están disponibles desde el Domingo a las 5pm CT hasta el Viernes a las 5pm CT. Si pides una llamada fuera de este horario, te llamaremos de 9am a 5pm en tu horario local.',
+        availabilityText: '*Nuestros guías están disponibles de domingo desde las 17.00 CT a viernes hasta las 17.00 CT. Si solicitas una llamada fuera de este horario, te llamaremos de lunes a viernes de 9.00 a 17.00 en tu hora local. El orden de llamada corresponde con el orden en el que las llamadas son recibidas.',
```
Comment (HorusGoul) [inline src/languages/es.js:359]: I would change the last part to look like this:
```suggestion
        availabilityText: '*Nuestros guías están disponibles de domingo desde las 17.00 CT a viernes hasta las 17.00 CT. Si solicitas una llamada fuera de este horario, te llamaremos de lunes a viernes de 9.00 a 17.00 en tu hora local. El orden de llamada corresponde con el orden de solicitud.',
```

**Comment by jasperhuangg** [comment]
@HorusGoul Thanks for the fix! Should be good for another review, feel free to merge if you think everything checks out, thanks!!

File: src/languages/es.js [hunk 2]
```diff
@@ -352,4 +352,14 @@ export default {
         confirmCompanyIsNot: 'Confirmo que esta empresa no está en el',
         listOfRestrictedBusinesses: 'lista de negocios restringidos',
     },
+    requestCallPage: {
+        requestACall: 'Llámame por teléfono',
+        description: 'Necesitas ayuda configurando tu cuenta? Nuestro equipo de guías te pueden ayudar.',
```
Comment (HorusGoul) [inline src/languages/es.js:357]:
```suggestion
        description: '¿Necesitas ayuda configurando tu cuenta? Nuestro equipo de guías puede ayudarte.',
```

File: src/languages/es.js [hunk 3]
```diff
@@ -352,4 +352,14 @@ export default {
...
+        instructions: 'Escribe tu nombre y número de teléfono y te llamaremos.',
+        availabilityText: '*Nuestros guías están disponibles de domingo desde las 17.00 CT a viernes hasta las 17.00 CT. Si solicitas una llamada fuera de este horario, te llamaremos de lunes a viernes de 9.00 a 17.00 en tu hora local. El orden de llamada corresponde con el orden de solicitud.',
+        callMe: 'Llámame',
+        growlMessageOnSave: 'Llamada perdida.',
```
Comment (HorusGoul) [inline src/languages/es.js:361]:
```suggestion
        growlMessageOnSave: 'Llamada solicitada.',
```

File: src/languages/es.js [hunk 4]
```diff
@@ -352,4 +352,14 @@ export default {
...
+        growlMessageInvalidPhone: 'El teléfono no es valido. Intenta nuevamente agregando el código de país. Eg: +15005550006',
```
Comment (HorusGoul) [inline src/languages/es.js:362]:
```suggestion
        growlMessageInvalidPhone: 'El teléfono no es valido. Intentalo de nuevo agregando el código de país. P. ej.: +15005550006',
```

**Review by HorusGoul — CHANGES_REQUESTED** [review]
I missed a lot of stuff on my first review. These are all the things I'd change:

**Comment by jasperhuangg** [comment]
@HorusGoul Thanks again! Should be good for another review 👍

**Review by HorusGoul — APPROVED** [review]
LGTM!

**Pull request merged**